cmd/config/virt-clone/check.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+COMMAND=$1
+LABEL_KEY=$2
+LABEL_VALUE=$3
+NAMESPACE=$4
+IDENTITY_FILE=$5
+REMOTE_USER=$6
+EXPECTED_ROOT_SIZE=$7
+EXPECTED_DATA_SIZE=$8
+
+# Wait up to ~60 minutes
+MAX_RETRIES=130
+# In the first reties use a shorter sleep
+MAX_SHORT_WAITS=12
+SHORT_WAIT=5
+LONG_WAIT=30
+
+if virtctl ssh --help | grep -qc "\--local-ssh " ; then
+    LOCAL_SSH="--local-ssh"
+else
+    LOCAL_SSH=""
+fi
+
+get_vms() {
+    local namespace=$1
+    local label_key=$2
+    local label_value=$3
+
+    local vms
+    vms=$(kubectl get vm -n "${namespace}" -l "${label_key}"="${label_value}" -o json | jq .items | jq -r '.[] | .metadata.name')
+    local ret=$?
+    if [ $ret -ne 0 ]; then
+        echo "Failed to get VM list"
+        exit 1
+    fi
+    echo "${vms}"
+}
+
+remote_command() {
```

Comment:
Getting this merged to unblock you, but I think having some of these utilities in go-commons would be nice for re-usability.

Reply:
I'm not sure this is doable for adding shell scripts into the embedded filesystem. The way this currently works requires me to duplicate the script for each workload.
See https://github.com/kube-burner/kube-burner/pull/783 for a similar issue